### PR TITLE
Migrate to create-plugin

### LINKED ISSUE #167: Move to @grafana/create-plugin
Grafana toolkit is going to be deprecated and the way forward for plugins is to use the new create-plugin library. https://grafana.github.io/plugin-tools/

### PR DESCRIPTION
**Pull request by zoltanbedi**
Fixes #167

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
 
-const standard = require('@grafana/toolkit/src/config/jest.plugin.config');
-
-// This process will use the same config that `yarn test` is using
-module.exports = standard.jestConfig();
+module.exports = {
+  // Jest configuration provided by @grafana/create-plugin
+  ...require('./.config/jest.config'),
+  // Inform jest to only transform specific node_module packages.
+  transformIgnorePatterns: [nodeModulesToTransform([...grafanaESModules, 'rxjs'])],
```
Comment (zoltanbedi) [inline jest.config.js:8]: Had to add rxjs as well as it was trying to use an esm module.

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+16
```
Comment (baldm0mma) [inline .nvmrc:1]: Is this missing new line an issue?

File: .prettierrc.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
 module.exports = {
-  ...require('@grafana/toolkit/src/config/prettier.plugin.config.json'),
-};
+  // Prettier configuration provided by Grafana scaffolding
+  ...require("./.config/.prettierrc.js")
+};
```
Comment (baldm0mma) [inline .prettierrc.js:4]: Is this missing new line an issue?

**Review by baldm0mma — APPROVED** [review]
Looks good to me, just a few low-importance nit picks about ending with new lines. :)

**Pull request merged**